File: Carthage/Checkouts/SwiftTheme/OCDemo/SelectThemeCell.m
//
//  SelectThemeCell.m
//  SwiftTheme
//
//  Created by Gesen on 16/5/27.
//  Copyright © 2016年 Gesen. All rights reserved.
//

#import "SelectThemeCell.h"
#import "Global.h"
#import "OCDemo-Swift.h"

@implementation SelectThemeCell

- (void)awakeFromNib {
    [super awakeFromNib];
    
    self.title.theme_textColor = globalTextColorPicker;
    self.themeIcon.theme_image = [ThemeImagePicker pickerWithNames:@[@"icon_theme_red", @"icon_theme_yellow", @"icon_theme_blue", @"icon_theme_light"]];
}

@end

File: Carthage/Checkouts/SwiftTheme/Demo/SelectThemeCell.swift
//
//  SelectThemeCell.swift
//  Demo
//
//  Created by Gesen on 16/3/2.
//  Copyright © 2016年 Gesen. All rights reserved.
//

import UIKit

class SelectThemeCell: BaseCell {
    
    @IBOutlet weak var title: UILabel!
    @IBOutlet weak var themeIcon: UIImageView!

    override func awakeFromNib() {
        super.awakeFromNib()
        
        title.theme_textColor = globalTextColorPicker
        themeIcon.theme_image = ["icon_theme_red", "icon_theme_yellow", "icon_theme_blue", "icon_theme_light"]
    }

}
